### PR TITLE
Fixed OSX Project

### PR DESCRIPTION
**Pull request by IkeBart**
I figured out a way to fix the OSX project so that it compiles again.  I made some changes to GlutHost.cpp to use some updates.  I also removed and re-added the libmoai-osx target dependency for `src/aku/AKU.h` and `src/aku/AKU.cpp`.  In addition, I made some changes to the Xcode project files that might not even be necessary, but I made them before re-adding the target dependency.

### CODE REVIEW

File: src/hosts/GlutHost.cpp [hunk 1]
```diff
@@ -399,8 +399,8 @@ void GlutRefreshContext () {
 	#endif
 	
 	#ifdef GLUTHOST_USE_LUAEXT
-		AKUExtLoadLuacrypto ();
-		AKUExtLoadLuacurl ();
+		//AKUExtLoadLuacrypto ();
+		//AKUExtLoadLuacurl ();
```
Comment (kstenerud) [inline src/hosts/GlutHost.cpp]: Why are these commented out?

**Comment by IkeBart** [comment]
I guess I forgot to remove the commented code.

**Comment by aab29** [comment]
(Please do remove it.)  :smiley_cat:

**Comment by kstenerud** [comment]
:shipit:

File: xcode/libmoai/libmoai.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -7668,6 +7676,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 5751E6D118565F8B00775234 /* libmoai-osx-release.xcconfig */;
 			buildSettings = {
+				ARCHS = "$(ARCHS_STANDARD_32_BIT)";
```
Comment (NachoSoto) [inline xcode/libmoai/libmoai.xcodeproj/project.pbxproj]: 32 bits? I thought we were compiling with 64 bits already?

Comment (NachoSoto) [inline xcode/libmoai/libmoai.xcodeproj/project.pbxproj]: Also, this is already defined in the `.xcconfig` files, why not change it there instead in any case?

**Comment by tonidero** [comment]
I just removed the additions to the ARCHS and SDKROOT to the pbxproj file so it takes the `.xcconfig` files. If you want a 32 bit version of MOAI (it might be necessary for the MoaiSample to work) just change it in the `base-mac.xcconfig` file

**Comment by NachoSoto** [comment]
:+1: thanks @tonidero!

**Comment by aab29** [comment]
Is this ready then?

**Comment by IkeBart** [comment]
I think this pull request is ready to merge.  As it is currently set up, all I have to do is change the value of ARCHS in `base-mac.xcconfig` to `"$(ARCHS_STANDARD_32_BIT"` for the OS X project to compile without linker errors.

**Comment by aab29** [comment]
OK, one last call, and then I'm going to merge.  :ship:

**Comment by NachoSoto** [comment]
:shipit:

**Comment by IkeBart** [comment]
Please, do merge it.

**Pull request merged**